.env.pilotstaging

```diff
@@ -12,21 +15,21 @@ BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
 BLOCKSCOUT_WEB_DOCKER_IMAGE_TAG="web-f6c3e0888d1d0ef72dc8bf870808702b7fd13730"
 BLOCKSCOUT_INDEXER_DOCKER_IMAGE_TAG="indexer-f6c3e0888d1d0ef72dc8bf870808702b7fd13730"
 BLOCKSCOUT_WEB_REPLICAS=3
-BLOCKSCOUT_DB_SUFFIX="3"
+BLOCKSCOUT_DB_SUFFIX="4"
```

Comment:
Looking at the SQL db in GCP, this should be 5. I think we had to destroy and redeploy twice